### ISSUE TITLE
Line break formulas in comments cause page freezes

### ISSUE THREAD

**Issue by obiscr** [apache/incubator-answer-plugins#245]
Hey, I found an interesting bug.

If I enter a formula with line breaks in a comment, the page will freezes, and for others as well.

<img width="786" alt="image" src="https://github.com/user-attachments/assets/e90e6264-746a-4212-a26d-380d35aac249">

**In fact, the moment I clicked Add Comment, the page was already frozen.**

If anyone else visits this page, they will not be able to do anything on this page.

After a while, the browser showing a dialog

<img width="448" alt="image" src="https://github.com/user-attachments/assets/419611b2-14b5-40a6-a419-b6573f4406c8">

I checked the database, the data shows this.

<img width="782" alt="image" src="https://github.com/user-attachments/assets/7cd365e6-6fba-4f6d-9de6-808016b0eb3a">

**Comment by robinv8** [comment]
I'll follow up.

**Comment by robinv8** [comment]
I can't reproduce it, can you please tell me your steps in detail?

**Comment by obiscr** [comment]
Ah, I forgot to mention that.

I've installed this plugins:

```
./answer_original build \
--with github.com/apache/incubator-answer-plugins/connector-github \
--with github.com/apache/incubator-answer-plugins/connector-google \
--with github.com/apache/incubator-answer-plugins/connector-basic \
--with github.com/apache/incubator-answer-plugins/editor-formula \
--with github.com/apache/incubator-answer-plugins/embed-basic
```

Here is the version
<img width="1073" alt="image" src="https://github.com/user-attachments/assets/fadbd455-62de-407a-8bd3-05437646e020">

Can you try to reproduce after install these plugins.(Maybe the key is `editor-formula` plugin)

**Comment by obiscr** [comment]
BTW, I checked the browser logs and the server logs and I didn't find any errors

**Comment by robinv8** [comment]
I want to know what you typed into the editor that caused it to freeze.

**Comment by obiscr** [comment]
<img width="786" alt="image" src="https://github.com/user-attachments/assets/e90e6264-746a-4212-a26d-380d35aac249">

I mentioned it in the first image.

```
$$
\vec{P} = ma
$$
```

**Comment by robinv8** [comment]
Is it development mode?

**Comment by obiscr** [comment]
I'm not sure, how do I check if I'm in development mode?

**Comment by robinv8** [comment]
Sorry, I asked a question that was incorrect. I'll check it out first.

**Comment by sy-records** [comment]
Fixed in [v1.4.2-RC1](https://github.com/apache/incubator-answer/releases/tag/v1.4.2-RC1)
